manager/gui/src/components/dashboard/LrDashboardCapacityCard.vue

```diff
@@ -66,11 +77,11 @@ export default class LrDashboardCapacityCard extends Vue {
   public capacityDetails: CapacityData = {} as CapacityData;
 
   public async mounted() {
-    const data = await Api.getData("/dashboard/capacity", {
+    const data: any = await Api.getData("/dashboard/capacity", {
```

Comment:
'data' can be renamed as 'res' or 'response'. Coz in the next line, it is called "data.data".

Reply:
Changed